web/js/natural-events/track.js

```diff
@@ -203,6 +207,33 @@ export default function naturalEventsTrack(models, ui, config) {
   init();
   return self;
 }
+/**
+ * Determine if track point is across the date line from the
+ * selected point
+ * @param {Array} activeCoord Coordinate array of selected track point
+ * @param {Array} nextCoord Coordinate to test against active coord
+ * @return {Boolean}
+ */
+const crossesDateLine = function(activeCoord, nextCoord) {
+  return (
+    Math.abs(Math.abs(activeCoord[0]) - Math.abs(nextCoord[0])) <
```

Comment:
@edplato  @mike-mcgann is there a better way to do this?

Reply:
One way would to assume that no two points would be separated by more than 180 degrees. Assume cross if abs(x1 - x2) > 180. 

I don't think what you have works when x1 = -1 and x2 = 1:

- abs(abs(-1) - abs(1)) < abs(-1 - 1) 
- abs(1 - 1) < abs(-2) 
- 0 < 2
- true

Reply:
it seems to work fine for me with the tropical storm in the mock data, but Mike's 
`Math.abs(activeCoord[0] - nextCoord[0]) > 180` may be easier to debug

Reply:
Good call. Thanks y'all.